### PR TITLE
v028 build 0

### PR DESCRIPTION
**Pull request by ofajardo**
Checklist
* [x ] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Reset the build number to `0` (if the version changed)
* [x] Ensured the license file is being packaged.

On windows compilation changed from mvsc to mingw

### CODE REVIEW

**Comment by ofajardo** [comment]
hi @isuruf I am having problems with this PR: windows is failing with the error: Unsatisfiable dependencies for platform win-64: {'m2w64_c_win-64'} (while my other feedstock is working fine (https://github.com/conda-forge/pyreadstat-feedstock/runs/612812098 ... I will delete this branch, just leave it there for now to demonstrate it works).
I basically copied the recipe from my other feedstock so I have no clue why it works over there but not here. 

Any suggestion?

thanks!

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,39 +1,45 @@
...
   build:
-    - {{ compiler('c') }}
+    - {{ compiler("m2w64_c") }}  # [win]
+    - {{ compiler('c') }}  # [not win]
   host:
     - python
     - pip
     - cython
-    - zlib
+    - m2w64-toolchain  # [win]
```
Comment (isuruf) [inline recipe/meta.yaml:25]: Remove this

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,39 +1,45 @@
...
+    - libpython  # [win]
+    - zlib  # [not win]
+    - bzip2  # [not win]
   run:
     - python
     - pandas
-    - zlib
+    - m2w64-gcc-libs  # [win]
```
Comment (isuruf) [inline recipe/meta.yaml:32]: Same here

**Comment by isuruf** [comment]
@conda-forge-admin, rerender

**Comment by ofajardo** [comment]
@isuruf thanks a bunch! I removed what you suggested. Windows works!
I guess I just wait a bit for OSX to be fixed and then trigger another build?

**Pull request merged**